### PR TITLE
Style app of the week cluster

### LINKED ISSUE #59: [Frontend] Styling for App of the Week Cluster

### PR DESCRIPTION
**Pull request by essencec**
Screenshot 2020-08-05 at 12 05 29 PM

### CODE REVIEW

File: client/src/Clusters/AppOfTheWeek/AppOfTheWeek.js [hunk 1]
```diff
@@ -9,15 +11,40 @@ class AppOfTheWeek extends Component {
...
+                        </div>
+                        <div className="content">
+                            <h3 className="app-name">{data.card.app.title}</h3> 
+                            <p className="developer">{data.card.app.developer}</p>
+                        </div>
+                    </div>
+                    <div className="reviewContainer">
+                        <div className="review-content">
+                            <p className="rating">
+                            {data.card.app.rating}
```
Comment (tonymai) [inline client/src/Clusters/AppOfTheWeek/AppOfTheWeek.js]: indent content inside the <p> element

File: client/src/Clusters/AppOfTheWeek/AppOfTheWeek.js [hunk 2]
```diff
@@ -9,15 +11,40 @@ class AppOfTheWeek extends Component {
...
+                            <img 
+                            className="rating-star-icon"
```
Comment (tonymai) [inline client/src/Clusters/AppOfTheWeek/AppOfTheWeek.js]: indent subsequent lines of a multiline tag as well

**Review by tonymai — DISMISSED** [review]
add a screenshot of the UI in your PR description

File: client/src/Clusters/AppOfTheWeek/AppOfTheWeek.style.css [hunk 3]
```diff
@@ -1,21 +1,116 @@
...
-    width: 80px;
-    border-radius: 20px;
+    border-radius: 12px;
+    bottom: 16.67%;
+    left: 0%;
+    margin-bottom: 30px;
+    margin-right: 16px;
+    position: relative;
+    right: 84.91%;
+    top: 16.67%;
```
Comment (ajwang0) [inline client/src/Clusters/AppOfTheWeek/AppOfTheWeek.style.css]: Is the top, bottom, left, right needed?

File: client/src/Clusters/AppOfTheWeek/AppOfTheWeek.style.css [hunk 4]
```diff
@@ -1,21 +1,116 @@
...
+}
+
+.app-of-the-week .review-content .rating-star-icon {
+    margin-left: 3px;
+}
+
+.app-of-the-week .review-content .reviews {
+    font-size: 12px;
+    line-height: 16px;
+    margin-top: 5.5px;
```
Comment (ajwang0) [inline client/src/Clusters/AppOfTheWeek/AppOfTheWeek.style.css]: Can we use whole numbers. we can round up

File: client/src/Clusters/AppOfTheWeek/AppOfTheWeek.style.css [hunk 5]
```diff
@@ -1,21 +1,116 @@
...
+.app-of-the-week .rated-content .rated {
+    font-size: 12px;
+    line-height: 16px;
+    margin-right: 30px;
+}
+
+.app-of-the-week .line {
+    border-right: 1px solid black;
+    margin-left: 25.67px;
+    margin-right: 24.33px;
```
Comment (ajwang0) [inline client/src/Clusters/AppOfTheWeek/AppOfTheWeek.style.css]: we can just use margin: 0 25px;

File: client/src/Clusters/AppOfTheWeek/AppOfTheWeek.style.css [hunk 6]
```diff
@@ -1,21 +1,116 @@
...
+}
+
+.app-of-the-week .button {
+    margin-right: 64.66px;
```
Comment (ajwang0) [inline client/src/Clusters/AppOfTheWeek/AppOfTheWeek.style.css]: Can we use whole numbers. we can round up

File: client/src/App.js [hunk 7]
```diff
@@ -16,35 +16,38 @@ class App extends Component {
...
-        })}
-        <Footer />
-      </div>
+        <div className='app'>
+            <Header />
+            <div className='clusters'>
+                {this.state.streamData.clusters.map((cluster) => {
+                    switch (cluster.type) {
+                        case 'AppOfTheWeekCluster':
+                            return <AppOfTheWeek data = {cluster} key = {cluster.id} />
```
Comment (ajwang0) [inline client/src/App.js]: Can you remove the spaces inbetween the prop key and the value

`data={cluster} key={cluster.id}`

**Review by ajwang0 — CHANGES_REQUESTED** [review]

**Comment by essencec** [comment]
Updated screenshot
image

**Review by ajwang0 — APPROVED** [review]

**Review by tonymai — APPROVED** [review]

**Pull request merged**